projects/kit/components/input-phone-international/input-phone-international.component.ts

```diff
@@ -111,6 +110,10 @@ export class TuiInputPhoneInternationalComponent
         private readonly options: TuiInputPhoneInternationalOptions,
         @Inject(TuiFlagPipe)
         private readonly flagPipe: TuiFlagPipe,
+        @Inject(TuiExtractCountryCodePipe)
```

Comment:
I'm against adding providers in modules, that's why I suggested utility functions. But in this case we can just add pipes to providers of this component and inject them. This way we retain DI hierarchy of `TUI_COUNTRIES_MASKS`,